WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -461,6 +469,25 @@ private extension OrderDetailsDataSource {
         cell.hideSeparator()
     }
 
+    private func configureShippingLabelCreationInfo(cell: ImageAndTitleAndTextTableViewCell, showsSeparator: Bool) {
+        cell.update(with: .imageAndTitleOnly(fontStyle: .footnote),
+                    data: .init(title: Title.shippingLabelCreationInfoAction,
+                                image: .infoOutlineFootnoteImage,
+                                imageTintColor: .systemColor(.secondaryLabel),
+                                numberOfLinesForTitle: 0,
+                                isActionable: false,
+                                showsSeparator: showsSeparator))
+
+        cell.selectionStyle = .default
+
+        cell.accessibilityTraits = .button
+        cell.accessibilityLabel = Title.shippingLabelCreationInfoAction
+        cell.accessibilityHint =
+            NSLocalizedString("Tap to show instructions on how to create a shipping label on the mobile device",
```

Comment:
Maybe it is superfluous to specify “on mobile device”?

Reply:
Makes sense! Updated in https://github.com/woocommerce/woocommerce-ios/pull/3769/commits/0125fea0e2b728d7ba5b1fb19a7ed1192a689d01